src/main/java/io/vertx/mqtt/impl/MqttServerImpl.java

```diff
@@ -85,8 +80,34 @@ public MqttServer listen(Handler<AsyncResult<MqttServer>> listenHandler) {
   public MqttServer listen(int port, String host, Handler<AsyncResult<MqttServer>> listenHandler) {
     Handler<MqttEndpoint> h1 = endpointHandler;
     Handler<Throwable> h2 = exceptionHandler;
-    Handler<MqttConnection> mqttConnectionHandler = c -> c.init(h1, h2);
-    listen(mqttConnectionHandler, port, host, ar -> listenHandler.handle(ar.map(this)));
+    server.connectHandler(so -> {
+      NetSocketInternal soi = (NetSocketInternal) so;
+      soi.messageHandler(o -> {});
```

Comment:
why the need for setting an "empty" handler here when it will be set later (line 103) ?